### ISSUE TITLE
Images not working

### ISSUE THREAD

**Issue by RactivussDevs** [optilude/xlsx-template#135]
Hi! I'm using version 1.3.0 of xlsx-template but I can't make images work. This is my .docx template:

| My image: | ${image:imageName} |

Just like in the example, and this is my code:

const data = {
imageName: //... buffer
};

I already try with a 'base64' image or an absolute path, but nothing.

Thanks in advance.

**Comment by CSMcrew** [comment]
Having same issue: 
    "xlsx-template": "1.3.0"
    Node.js v14.16.0.
    
In the demo project local  version is used:
var XlsxTemplate = require('../lib'), and it differs with  "1.3.0" from NPM

RactivussDevs, did you solve your problem ?

**Comment by RactivussDevs** [comment]
No, I couldn't solve it. And I use a .docx template named as 'template.docx'. I'll try that '../lib' import and see what happens.

**Comment by enmanuel-mag** [comment]
Same problem:
"xlsx-template": "1.3.0"
Nodejs v14.15.0

I tried with Buffer, Base64 and doesn't working

**Comment by CSMcrew** [comment]
> Same problem:
> "xlsx-template": "1.3.0"
> Nodejs v14.15.0
> 
> I tried with Buffer, Base64 and doesn't working

Still there ((

**Comment by kant2002** [comment]
Hi guys. Seems to be I wake up from hybernation. From the discussion it is not clear, are you using `.docx` format and not `.xlsx` files? We do not support template for Word documents.

**Comment by IllusionVK** [comment]
Hey @AyngDevs (or @CSMcrew) Were you able to check if you were using .docx or .xlsx ?

**Comment by smolleyes** [comment]
i use xlsx and not working too buffer/path/base64 nothing work :/

**Comment by jdugh** [comment]
Hi everyone,
If you want to compare your code :
You can find an example here : https://github.com/optilude/xlsx-template/blob/master/test/crud-test.ts#L1119-L1130
And the .xlsx file here : https://github.com/optilude/xlsx-template/blob/master/test/templates/test-insert-images.xlsx

This is an XLSX file (and not a DOCX) !
Test working, and you use xlsx-template every day with image insertion. All work fine.

Another test with buffer, base64 string, and static image : 
https://github.com/optilude/xlsx-template/blob/master/test/crud-test.ts#L1228-L1252
